CHANGELOG.md

```diff
@@ -1,9 +1,8 @@
-## 3.4.0(unreleased)
+## 3.4.0
 ### Notes
 - This release extends support of the SDK to OneView REST API version 1600 (OneView v5.20) and (ImageStreamer v5.20).
```

Comment:
remove this " and (ImageStreamer v5.20)"

Reply:
Updated.